src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java

```diff
@@ -15,18 +18,30 @@
 public class DriverCommandManager {
     private DriverCommand currentCommand = null;
     private Publisher changePublisher = new Publisher();
-    private final CommandVisitor<Map<String, Integer>> countingVisitor = new CommandCounterVisitor();
-    private final CommandVisitor<Boolean> boundaryVisitor = new CommandBoundaryVisitor();
+    private final List<CommandVisitorSubscriber> visitorSubscribers = new ArrayList<>();
+    // Constructor to add default subscribers
+    public DriverCommandManager() {
+        visitorSubscribers.add(new BoundaryVisitorSubscriber());
+        visitorSubscribers.add(new CounterVisitorSubscriber());
```

Comment:
Boundary and Counter visitors should be added externally (to avoid dependencies)